client/src/App.jsx

```diff
@@ -4,14 +4,14 @@ import {
   Routes,
   Route,
   // Redirect;
-} from 'react-router-dom';
-import LandingPage from './pages/LandingPage';
-import OnboardPage from './pages/OnboardPage';
-import Redirect from './pages/LandingPage/Redirect';
-import ProfilePage from './pages/ProfilePage/ProfilePage';
-import RequestDashboardPage from './pages/RequestDashboardPage';
-import OpeningDashboardPage from './pages/OpeningDashboardPage/OpeningDashboardPage';
-import RequestCreatePage from './pages/RequestCreatePage';
+} from "react-router-dom";
+import LandingPage from "./pages/LandingPage";
+import OnboardPage from "./pages/OnboardPage";
+import Redirect from "./pages/LandingPage/Redirect";
+import ProfilePage from "./pages/ProfilePage/ProfilePage";
+import RequestDashboardPage from "./pages/RequestDashboardPage";
+import OpeningDashboardPage from "./pages/OpeningDashboardPage/OpeningDashboardPage";
+import RequestCreatePage from "./pages/RequestCreatePage";
```

Comment:
should setup some sort of repository-level prettier setting to avoid these kinds of diffs